assemblies/rosa-get-started-cli.adoc

```diff
@@ -0,0 +1,18 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+[id="rosa-get-started-cli"]
+
+= Getting started with the `rosa` CLI
```

Comment:
Per recent discussion, no formatting in titles

Reply:
Fixed.